src/EditorFeatures/Test2/Simplification/ParenthesisSimplificationTests.vb

```diff
@@ -1042,346 +1297,279 @@ using System;
-    End Sub
-End Class
+    void M()
+    {
+        var x = "" + {|Simplify:(new C() + new C())|};
+    }
+}
         </Document>
     </Project>
 </Workspace>
 
             Dim expected =
 <code>
-Imports System.Xml.Linq
+class C
+{
+    public static string operator +(C left, C right) => "";
 
-Class C
-    Sub M()
-        Dim x = 1 &gt; (&lt;xml/&gt;)
-    End Sub
-End Class
+    void M()
+    {
+        var x = "" + (new C() + new C());
+    }
+}
 </code>
 
             Await TestAsync(input, expected)
         End Function
 
-        <WorkItem(4490, "https://github.com/dotnet/roslyn/issues/4490")>
+        <WorkItem(11958, "https://github.com/dotnet/roslyn/issues/11958")>
         <Fact, Trait(Traits.Feature, Traits.Features.Simplification)>
-        Public Async Function TestVisualBasic_DontRemoveParenthesesAroundXmlElementWhenPreviousTokenIsLessThan() As Task
+        Public Async Function TestCSharp_SimplifyIfBinaryExpressionTypeIsIdentityConversion() As Task
             Dim input =
 <Workspace>
-    <Project Language="Visual Basic" CommonReferences="true">
+    <Project Language="C#" CommonReferences="true">
         <Document>
-Imports System.Xml.Linq
-
-Class C
-    Sub M()
-        Dim x = 1 &lt; {|Simplify:(&lt;xml&gt;&lt;/xml&gt;)|}
-    End Sub
-End Class
+class C
+{
+    int Add(int a, int b, int c) => a + {|Simplify:(b + c)|};
+}
         </Document>
     </Project>
 </Workspace>
 
             Dim expected =
 <code>
-Imports System.Xml.Linq
-
-Class C
-    Sub M()
-        Dim x = 1 &lt; (&lt;xml&gt;&lt;/xml&gt;)
-    End Sub
-End Class
+class C
+{
+    int Add(int a, int b, int c) => a + b + c;
+}
 </code>
 
             Await TestAsync(input, expected)
         End Function
 
-        <WorkItem(4490, "https://github.com/dotnet/roslyn/issues/4490")>
+        <WorkItem(11958, "https://github.com/dotnet/roslyn/issues/11958")>
         <Fact, Trait(Traits.Feature, Traits.Features.Simplification)>
-        Public Async Function TestVisualBasic_DontRemoveParenthesesAroundXmlElementWhenPreviousTokenIsGreaterThan() As Task
+        Public Async Function TestCSharp_SimplifyIfBinaryExpressionTypeIsImplicitNumericConversion() As Task
             Dim input =
 <Workspace>
-    <Project Language="Visual Basic" CommonReferences="true">
+    <Project Language="C#" CommonReferences="true">
         <Document>
-Imports System.Xml.Linq
-
-Class C
-    Sub M()
-        Dim x = 1 &gt; {|Simplify:(&lt;xml&gt;&lt;/xml&gt;)|}
-    End Sub
-End Class
+class C
+{
+    int Add(int a, short b, short c) => a + {|Simplify:(b + c)|};
+}
         </Document>
     </Project>
 </Workspace>
 
             Dim expected =
 <code>
-Imports System.Xml.Linq
-
-Class C
-    Sub M()
-        Dim x = 1 &gt; (&lt;xml&gt;&lt;/xml&gt;)
-    End Sub
-End Class
+class C
+{
+    int Add(int a, short b, short c) => a + b + c;
+}
 </code>
 
             Await TestAsync(input, expected)
         End Function
+
+#End Region
```

Comment:
The tests in this file had been organized at one point, but recently added tests weren't added with the same organization. I organized them.